webrtc.html

```diff
@@ -4166,7 +4166,7 @@ <h2 id="sec.stats-model">Statistics Model</h2>
 
       <dl class="idl" title="partial interface RTCPeerConnection">
         <dt>Promise&lt;RTCStatsReport&gt; getStats(
-          optional MediaStreamTrack? selector)</dt>
+          optional MediaStreamTrack? selector = null)</dt>
```

Comment:
> MediaStreamTrack? selector = null

`WebIDL.WebIDLError: error: Mandatory arguments can't have a default value.`

Unlike in C++, a WebIDL default does not implicitly make a parameter optional apparently.

Having an explicit `null` overload for "select everything" makes some sense I suppose (if one buys into that `null` vs. `undefined` is useful for anything at all in the JS language).